ethereum/graphqlrpc/src/test/resources/tech/pegasys/pantheon/ethereum/graphqlrpc/graphql_pending.json

```diff
@@ -0,0 +1,22 @@
+{
+  "request":
+    "{ pending { transactionCount transactions { from { address } to { address } } account(address:\"0x6295ee1b4f6dd65047762f924ecd367c17eabf8f\") { balance} estimateGas(data:{}) call (data : {from : \"a94f5374fce5edbc8e2a8697c15331677e6ebf0b\", to: \"0x6295ee1b4f6dd65047762f924ecd367c17eabf8f\", data :\"0x12a7b914\"}){data status}} }",
+
+  "response": {
+    "data": {
+      "pending": {
+        "transactionCount": 0,
+        "transactions": [],
```

Comment:
Can we add some more tests that return non-empty tx's here?

Reply:
mocked in a pending transaction that will return nonce and gas.